src/firecracker/Cargo.toml

```diff
@@ -30,3 +30,31 @@ vmm = { path = "../vmm" }
 [dev-dependencies]
 cargo_toml = "0.15.3"
 regex = { version = "1.8.4", default-features = false, features = ["std"] }
+
+# Dev-Dependencies for uffd examples
+serde = { version = "1.0.165", features = ["derive"] }
+userfaultfd = "0.6.0"
+
+[[example]]
+name = "uffd_malicious_handler"
+path = "../../examples/uffd/malicious_handler.rs"
```

Comment:
is it idiomatic to use relative paths here?

Reply:
If the examples where under `src/firecracker/examples` they would be linked automatically. I think this is the simplest approach instead of using paths.

Reply:
They still wouldn't get picked up automatically since they are in subdirectories, and they'd only get picked up automatically if there was one main function per subdir. Can move them to src/firecracker though if that's preferred, then the paths wouldn't have "../" components.

Reply:
done

Reply:
I don't have a preference here, no much difference between `../` and `../../`. @JonathanWoollett-Light what do you think?

Reply:
It'd be just `examples/bla/bla/bla` if its inside firecracker. I actually kinda prefer it to be not a top level directory the more I think about it - its not like its "production grade" code and shouldn't be this prominent

Reply:
I think it is unfortunate we need to specify example paths, I think I would prefer if we could use a prefix like `uffd_` or `seccomp_` to avoid needing to specify paths, but this is a nit.